README.md

```diff
@@ -125,6 +125,7 @@ ___
             hookRenderRoute: true, // Activate the prune during the `hook:render:route`
             hookGeneratePage: true, // Activate the prune during the `hook:generate:page`
             lighthouseUserAgent: 'lighthouse', // Value of the Lighthouse UserAgent, either as String or RegExp (a string will be converted to a case-insensitive RegExp in the MobileDetect library)
+            headerOverrideName: 'user-agent', // Value of a custom header name passed from a Lambda Edge function, or similar
```

Comment:
We should simply call it **headerName**, because _override_ is implied if you pass a _new-configuration_.